hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -175,23 +188,94 @@ public static Set<Path> normalizeExcludeList(List<String> toExcludeList,
     return paths;
   }
 
+  @Override
+  // Pauses rocksdb compaction threads while creating copies of
+  // compaction logs and hard links of sst backups.
+  public DBCheckpoint getCheckpoint(Path tmpdir, boolean flush)
+      throws IOException {
+    DBCheckpoint checkpoint;
+
+    // make tmp directories to contain the copies
+    RocksDBCheckpointDiffer differ = getDbStore().getRocksDBCheckpointDiffer();
+    DirectoryData sstBackupDir = new DirectoryData(tmpdir,
+        differ.getSSTBackupDir());
+    DirectoryData compactionLogDir = new DirectoryData(tmpdir,
+        differ.getCompactionLogDir());
+    if (!sstBackupDir.getTmpDir().mkdirs()) {
+      throw new IOException("mkdirs failed: " + sstBackupDir.getTmpDir());
+    }
```

Comment:
nit: this can be move to the `DirectoryData` constructor.

Reply:
done